sdk/keyvault/azsecrets/ci.yml

```diff
@@ -0,0 +1,17 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
```

Comment:
Lets please also add in the branch triggers. See recent updates to other ci.yml files where @benbp  added these.